src/t8_schemes/t8_default/t8_default_tet/t8_default_tet_cxx.cxx

```diff
@@ -564,5 +564,14 @@ t8_default_scheme_tet_c::~t8_default_scheme_tet_c ()
    * However we need to provide an implementation of the destructor
    * and hence this empty function. */
 }
-
+void
```

Comment:
There is no need to implement it here, if you use the `T8_DTRI_TO_DTET` logic in the triangle implementation